modules/logic.js

```diff
@@ -0,0 +1,81 @@
+const allBooks = document.querySelector('.all-books');
+const form = document.querySelector('.form');
+const [title, author] = form.elements;
+
+const inputBook = {};
+// eslint-disable-next-line import/no-mutable-exports
+let books = new Array([]);
```

Comment:
- [ ] Let's not disable linter checks as they make us write better codes. I suggest you use the array literal method 
`[] `  for simplicity, readability, and execution speed as there is no need to use `new Array()` method. See my suggestion below:

```suggestion
let books = [];
```

Reply:
![image](https://user-images.githubusercontent.com/67393791/150508009-3001c6b0-caab-4a97-a456-83c809565047.png)


If I follow this suggestion, it trow me an error. Honestly, I don't feel like you want to block me. But you definitively are. Please reconsider your review. My app was working flawless. Now is all broke. And I got no approval for this project.